components/SidePage/SidePage.module.css

```diff
@@ -1,6 +1,6 @@
 .SidePage__container {
     height: 100vh;
-    width: max(350px, 16%);
+    width: max(var(--main-page-container-width), var(--offset));
```

Comment:
Let's use classic `max-width` and `width` combination. It's more readable

```
width: 350px;
max-width: 16%;
```

Reply:
It doesn't work the same